README.md

```diff
@@ -20,59 +20,74 @@ For a full description of request and response payloads and properties, please s
 
 ## Installation
 
-Add this line to your go package:
+Download the package using `go get`:
 
 ```bash
-import "github.com/trycourier/courier-go"
+go get -u github.com/trycourier/courier-go/v2
 ```
 
 ## Usage
 
 ```go
-func send() {
-        var eventID = "example-event"
-        var recipientID = "example-recipient"
-
-        client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
-        requestID, err := client.SendMessage(
-                context.Background(),
-                courier.SendMessageRequestBody{
-                        "template": "<COURIER_TEMPLATE>",
-                        "to": map[string]string{
-                                "email": "test@email.com"
-                        }
-                }
-        )
-        if err != nil {
-                log.Fatalln(err)
+package main
+
+import (
+	"context"
+	"fmt"
+	
+	"github.com/trycourier/courier-go/v2"
+)
+
+func main() {
+	client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
+	message := courier.SendMessageRequestBody{
+		"template": "<COURIER_TEMPLATE>",
+			"to": map[string]string{
+				"email": "test@email.com"
+		}
 	}
-        log.Println(requestID)
+
+	id, err := client.SendMessage(context.Background(), message)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(id)
 }
 ```
 
 If you would like to send an idempotent message then be sure to use the SendMessageWithOptions method as shown below:
 
 ```go
-func sendIdempotentMessage() {
-        var eventID = "example-event"
-        var recipientID = "example-recipient"
-        expiration := time.Now().Add(time.Hour * 30)
-        client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
-        requestID, err := client.SendMessageWithOptions(
-                context.Background(),
-                map[string]interface{}{
-                        "template": "<COURIER_TEMPLATE>",
-                        "to": map[string]string{
-                                "email": "test@email.com"
-                        }
-                },
-                courier.WithIdempotencyKey("fake-key"),
-		courier.WithIdempotencyKeyExpiration(expiration)
-        )
-        if err != nil {
-                log.Fatalln(err)
+package main
+
+import (
+	"fmt"
+	"time"
+	"context"
+	
+	"github.com/trycourier/courier-go/v2"
+)
+
+func main() {
+	client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
+	message := map[string]interface{}{
+		"template": "<COURIER_TEMPLATE>",
+			"to": map[string]string{
+				"email": "test@email.com"
+			}
 	}
-        log.Println(requestID)
+	id, err := client.SendMessageWithOptions(
```

Comment:
```suggestion
	reqId, err := client.SendMessageWithOptions(
```

Reply:
I think it should be clear from the code and Courier documentation that sending a message request will return a request ID.

Reply:
I agree but ID here feels really vague. I'd rather err on the side of being more verbose so that it leads to less confusion.

Reply:
In that case I'd recommend changing it to `reqID` (fully uppercase) at least, to comply with Go naming conventions. See: [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Oh yes, I like that!